marker_api/src/ast/common/span.rs

```diff
@@ -570,3 +570,31 @@ impl_ident_eq_for!(
     std::ffi::OsString,
     std::borrow::Cow<'_, str>
 );
+
+/// A trait for nodes, that provide a [`Span`].
```

Comment:
What do you think about implementing this trait on `Span` type itself?

Reply:
Yes, sounds good :)